src/docs/docsite.json

```diff
@@ -73,6 +73,7 @@
     "notes-1.16.x": "dist/markdown/html/src/python/pants/notes/1.16.x.html",
     "notes-1.17.x": "dist/markdown/html/src/python/pants/notes/1.17.x.html",
     "notes-1.18.x": "dist/markdown/html/src/python/pants/notes/1.18.x.html",
+    "notes-1.19.x": "dist/markdown/html/src/python/pants/notes/1.19.x.html",
```

Comment:
For future reference: in order for this page to be rendered on the left hand side of the site, it needs to be added to the Release notes section on line 201 in this file.